src/ext/VSExtension/wixext/VSHeatExtension.cs

```diff
@@ -153,6 +156,15 @@ public override void ParseOptions(string type, string[] args)
                         bool found = false;
                         foreach (string availableOutputGroup in allOutputGroups)
                         {
+                            // Allow for specifying the name along with a Metadata filter that can be applied to include/exclude items
+                            // The syntax for that will be "{pogName}:{Metadata Filter Expression}"
+                            // eg.  References:CopyLocal=true   or References:CopyLocal!=true
+                            Match expressive = Regex.Match(pogName, "([^:]+):(.*)", RegexOptions.IgnoreCase);
```

Comment:
Do you need to use Regex here? It seems like you are just looking for ':' and want to split on that?